modules/alloy/src/main/ts/ephox/alloy/api/ui/FloatingToolbarButton.ts

```diff
@@ -155,6 +155,9 @@ const factory: CompositeSketchFactory<FloatingToolbarButtonDetail, FloatingToolb
     },
     getToolbar(button: AlloyComponent) {
       return Sandboxing.getState(Coupling.getCoupled(button, 'toolbarSandbox'));
+    },
+    isOn(button: AlloyComponent) {
```

Comment:
Copying from Lee from Slack: `...Luke specifically used isOn as thats the same name the Toggling behaviour/API uses`.

Reply:
I am wondering if we should consider naming this differently now though, as `Toggling.isOn` is descriptive because of the behaviour name. Whereas `SplitFloatingToolbar.isOn` doesn't really make it clear what it means.

Reply:
Yeah, that's a good point. `isOpen` is definitely more descriptive